.prettierignore

```diff
@@ -14,8 +14,11 @@ apps/demo-typegen/src/script-types
 apps/demo-typegen/src/predicate-types
 apps/docs/.vitepress/cache/
 apps/create-fuels-counter-guide
-apps/docs-snippets/.fuels
-apps/docs-snippets2/.fuels
+apps/docs/.fuels
+apps/docs/.typedoc
+apps/docs/src/api
+apps/docs/src/**/*.ts
```

Comment:
This path is being ignored here but [added](https://github.com/FuelLabs/fuels-ts/pull/3369/files#diff-663ade211b3a1552162de21c4031fcd16be99407aae5ceecbb491a2efc43d5d2R29) in `.prettierrc`. I think one of them should go :smile:

Reply:
Great catch!

🦅  👀

Reply:
[9fbb033](https://github.com/FuelLabs/fuels-ts/pull/3369/commits/9fbb033a5ba0f9f788da5dcf5ffa165bc69bce07)

Reply:
I'd actually suggest ignoring at least the `test` files that get generated at least (potentially even the snippets as well)